custom field type and wrong decorator signature rule

borremosch opened the pull request:
Fixes #5

rchl commented:
Would be great if the rule would also document the default mappings that are enabled by default.

rchl commented:
And also whether defining custom mapping for a type that overlaps with default mappings will replace or just extend default one.

borremosch commented:
> Would be great if the rule would also document the default mappings that are enabled by default.

Added the default mappings to the docs!

> And also whether defining custom mapping for a type that overlaps with default mappings will replace or just extend the default one.

That is a good point, I have added and documented an optional flag that allows you to do either.
